app/scripts/controllers/paymentModal.js

```diff
@@ -39,6 +40,18 @@ angular.module('confRegistrationWebApp')
         return;
       }
 
+      if(permissions.permissionInt < permissionConstants.UPDATE){
+        if(permissions.permissionInt === permissionConstants.SCHOLARSHIP) {
+          if($scope.newTransaction.paymentType !== 'SCHOLARSHIP'){
+            alert('Your permission level only allows scholarship payments to be added.');
```

Comment:
Maybe add this phrase as part of your alert here too? 
"Please contact an event admin to request permission."